roles/openshift_repos/meta/main.yml

```diff
@@ -11,4 +11,5 @@ galaxy_info:
     - 7
   categories:
   - cloud
-dependencies: []
+dependencies:
+- role: openshift_inventory
```

Comment:
I see role dependency proliferation happening if we continue down this path (adding `openshift_inventory` as a dep in module meta files). Is there any avenue available for us to make this a standard pre-run action instead?